src/repo.ts

```diff
@@ -123,52 +125,6 @@ async function getFilesChangedSince(
   );
 }
 
-/**
- * Gets the HTTPS URL of the primary remote with which the given repository has
- * been configured. Assumes that the git config `remote.origin.url` string
- * matches one of:
- *
- * - https://github.com/OrganizationName/RepositoryName
- * - git@github.com:OrganizationName/RepositoryName.git
- *
- * If the URL of the "origin" remote matches neither pattern, an error is
- * thrown.
- *
- * @param repositoryDirectoryPath - The path to the repository directory.
- * @returns The HTTPS URL of the repository, e.g.
- * `https://github.com/OrganizationName/RepositoryName`.
- */
-export async function getRepositoryHttpsUrl(
```

Comment:
I split up this function into two pieces so that it could be tested better: I extracted the code to get the repository URL to `project.ts` and the string manipulation logic to `misc-utils.ts`.